build/Dockerfile

```diff
@@ -26,7 +27,8 @@ RUN mkdir -p /mattermost/data /mattermost/plugins /mattermost/client/plugins \
   else echo "please set the MM_PACKAGE" ; fi \
   && addgroup -g ${PGID} mattermost \
   && adduser -D -u ${PUID} -G mattermost -h /mattermost -D mattermost \
-  && chown -R mattermost:mattermost /mattermost /mattermost/plugins /mattermost/client/plugins
+  && chown -R mattermost:mattermost /mattermost /mattermost/plugins /mattermost/client/plugins \
+  && echo 'mattermost ALL=NOPASSWD: /usr/sbin/update-ca-certificates' >> /etc/sudoers
```

Comment:
Instead of adding `mattermost` user to `sudoers` file you can just directly execute `update-ca-certificates` and move it out of entrypoint.